test/unit/old-format.test.ts

```diff
@@ -1,12 +1,9 @@
 import { expect, test } from 'vitest';
 import * as policy from '../../lib';
 
-test('test sensibly bails if gets an old .snyk format', () => {
-  expect(() =>
-    policy.load(__dirname + '/../fixtures/old-snyk-config/').catch((e) => {
-      expect(e.message).toBe('old, unsupported .snyk format detected');
```

Comment:
An error was still being thrown after the commented out check, but this check did not catch it. Or rather, it did, but then re-threw on the check above (as the errors did not match) and the outer `toThrow` just checked that an error was thrown and not what the error was.

TLDR the test that should have caught the issue was also not working as expected